public/css/style.css

```diff
@@ -47,3 +47,31 @@ a.navbar-brand {
 select#languages {
     height: 30em;
 }
+
+h2 {
+	font-size: 1.3em;
+	font-weight: normal;
+	color: #000000;
+}
+h3 {
+	font-size: 1.2em;
+	font-weight: normal;
+	color: #000000;
+}
+
+h2.help {
+	font-size: 1.3em;
+	font-weight: normal;
+	color: #0c3762;
+	border-bottom: dotted thin #e0e0e0;
+	margin-top: 25px;
+}
+
+img.screenshot {
```

Comment:
Actually, remove this style. Wrap images in <p> so they get the same styling.

Reply:
OK. p.screenshot it is, adding <p.screenshot> around sreenshot images.